index.html

```diff
@@ -507,6 +380,10 @@ <h4>
         </li>
       </ul>
     </footer>
+    <section class="modal" id="modal">
```

Comment:
- [ ] When I click one of the project card, the popup project card doesn't display immediately, I will have to scroll up to view the popup card 

e.g 👇🏽 This is what I see when I open the popup unless I scroll up. Kindly fix this behavior so that the project card display without the need for scrolling 👍🏽 

![ezgif com-gif-maker (4)](https://user-images.githubusercontent.com/63473584/195449515-9502ebba-e5bc-48b5-8182-b47faf2baf88.gif)